resources/js/global.js

```diff
@@ -87,12 +106,26 @@ var Globals = {
   },
 
   // Function to validate an url
-  urlExists : function (url) {
+  urlExistsOLD : function (url) {
       var http = new XMLHttpRequest();
       http.open('HEAD', url, false);
       http.send();
       return http.status !== 404;
   },
+  
+  urlExists: function (url) {

```

Comment:
This was a test we put in on Tuesday night for the cross domain issue that didn't work, but it probably shouldn't be committed